framework/installation/pipelines/Templates/build-steps.yaml

```
@@ -3,22 +3,18 @@ parameters:
   useBeta: ''
   storageAccountName: ''
   storageSasToken: ''
+  version: '*'
 
 steps:
 - task: PowerShell@2
   displayName: 'Pull Sources'
   inputs:
     targetType: filePath
-    filePath: './deploy/Invictus-GetSources.ps1'
-    arguments: '-StorageAccountName "${{parameters.storageAccountName}}" -StorageSasToken "${{parameters.storageSasToken}}" -StorageContainerName "${{parameters.type}}" -SaveLocation "$(Build.ArtifactStagingDirectory)" -UseBeta ${{parameters.useBeta}}'
+    filePath: './Scripts/Invictus-GetSources.ps1'
```

```suggestion
    filePath: './scripts/Invictus-GetSources.ps1'
```
It was lowercase in the dashboard pipeline.